packages/components/src/components/event/eventInfo/EventsHierarchy.tsx

```diff
@@ -15,16 +6,31 @@ import {
 } from 'hds-react';
 import { useTranslation } from 'next-i18next';
 import React from 'react';
-
-import { useSubEvents, useSubEventsQueryVariables } from '../queryUtils';
+import { InfoWithIcon, LoadingSpinner, SkeletonLoader } from '../../../index';
+import type { EventFields, SuperEventResponse } from '../../../index';
+import type {
+  GetEventListLinkUrl,
+  UseSubEvents,
+  UseSubEventsQueryVariables,
+} from '../../../types';
 import EventList from './eventList/EventList';
 import styles from './eventList/eventList.module.scss';
 
 const EVENTS_LIST_LIMIT = 3;
 export const subEventsListTestId = 'sub-events-list';
 export const superEventTestId = 'super-event';
-
-const SubEvents: React.FC<{ event: EventFields }> = ({ event }) => {
+export interface Props {
+  event: EventFields;
+  useSubEvents: UseSubEvents;
+  useSubEventsQueryVariables: UseSubEventsQueryVariables;
+  getEventListLinkUrl: GetEventListLinkUrl;
+}
+const SubEvents: React.FC<Props> = ({
+  event,
+  useSubEvents,
+  useSubEventsQueryVariables,
+  getEventListLinkUrl,
+}) => {
```

Comment:
If I pass the query variables and the actual query hook separately, why wouldn't I add the variables inside the hook-function already? And if the variables can be passed in another prop, wouldn't it make sense to move the useSubEvents-hook to the components-package itself?